src/main/java/net/minecraftforge/registries/ForgeRegistry.java

```diff
@@ -101,6 +106,42 @@
     private final ResourceLocation name;
     private final RegistryKey<Registry<V>> key;
     private final RegistryBuilder<V> builder;
+    
+    private final Codec<V> CODEC = new Codec<V>() {
```

Comment:
This is not a `static` field.
```suggestion
    private final Codec<V> codec = new Codec<V>() {
```

Reply:
Yes, changed it to lowercase.